shared/common/ui/select/index.tsx

```diff
@@ -183,6 +185,7 @@ export function Select<T extends any>({
       className={cn(styles.select, className)}
       onClick={!disabled ? () => setDropdownOpen(true) : undefined}
       data-disabled={disabled}
+      data-testid={label}
```

Comment:
I was going to suggest doing the same `{..props}` thing as above so we're not embedding test specific stuff into what are meant to be generic ui components, but we already have `...dropdown`, so I guess this is ok for now to avoid too much refactoring.

Reply:
hm do u think renaming here dropdown to props will work? and then using sth like:  
`{...(props['data-testid'] && { 'data-testid': props['data-testid'] })}
`
and rename dropdown to props where needed.

Reply:
Actually refactoring to remove `...dropdown` isn't too bad, so I think this should work:
[edgedb-ui-10-44-16.patch](https://github.com/user-attachments/files/19387623/edgedb-ui-10-44-16.patch)